### PR TITLE
8.6.2 Release notes 

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Fixes https://github.com/elastic/security-docs/issues/2958.

Previews:
- [Index page](https://security-docs_2971.docs-preview.app.elstc.co/guide/en/security/master/release-notes.html)
- [Release notes](https://security-docs_2971.docs-preview.app.elstc.co/guide/en/security/master/release-notes-header-8.6.0.html#release-notes-8.6.2)

### CODE REVIEW

File: docs/release-notes/8.6.asciidoc [hunk 1]
```diff
@@ -1,6 +1,16 @@
...
 == 8.6
 
+[discrete]
+[[release-notes-8.6.2]]
+== 8.6.2
+
+[discrete]
+[[bug-fixes-8.6.2]]
+==== Bug fixes and enhancements
+* Fixes a bug that prevented related alerts from closing when the *Close all alerts that match this exception* option selected on an exception ({pull}150765[#150765]).
```
Comment (joepeeples) [inline docs/release-notes/8.6.asciidoc:11]:
```suggestion
* Fixes a bug that prevented related alerts from closing when the *Close all alerts that match this exception* option was selected on an exception ({pull}150765[#150765]).
```

**Review by dplumlee — APPROVED** [review]
Looks good!

**Review by benironside — APPROVED** [review]
LGTM, thanks!

**Review by jmikell821 — APPROVED** [review]
LGTM 👍

**Review by roxana-gheorghe — APPROVED** [review]

**Pull request merged**